generatebundlefile/hack/common.sh

```diff
@@ -29,7 +29,14 @@ function awsAuth () {
         region="--region=us-east-1"
     fi
 
-    aws "$awsCmd" "$region" "--profile=${PROFILE:-}" get-login-password
```

Comment:
Moving the oras login inside the function since we don't use the `awsAuth` function anywhere else.